draft-packages/menu/KaizenDraft/Menu/components/MenuList.tsx

```diff
@@ -1,29 +1,18 @@
-import React, { useState } from "react"
-import { v4 } from "uuid"
+import React, { HTMLAttributes } from "react"
+import { OverrideClassName } from "@kaizen/component-base"
 import styles from "./MenuList.module.scss"
 
-export type MenuListProps = {
-  heading?: React.ReactNode
+export interface MenuListProps
+  extends OverrideClassName<HTMLAttributes<HTMLUListElement>> {
   children: React.ReactNode
 }
```

Comment:
Would you want to make `aria-labelledBy` mandatory here?

Reply:
No because more often than not there's no heading